docs/source/usersguide/variance_reduction.rst

```diff
@@ -161,7 +161,7 @@ solver, the Python input just needs to load the h5 file::
 
     settings.weight_window_checkpoints = {'collision': True, 'surface': True}
     settings.survival_biasing = False
-    settings.weight_windows = openmc.WeightWindowsList.from_hdf5('weight_windows.h5')
+    settings.weight_windows_file = "weight_windows.h5"
```

Comment:
These are essentially equivalent lines, no?

Reply:
From the C++ side, they will have the same effect. The difference is that the `weight_windows_file` option won't put all the data in the model.xml file.